transports/quic/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.10.1 - unreleased
+
+- Allow disabling MTU discovering.
+  See [PR 4823].
+
+[PR 4823]: https://github.com/libp2p/rust-libp2p/pull/4823
```

Comment:
```suggestion
- Allow disabling path MTU discovery.
  See [PR 4823](https://github.com/libp2p/rust-libp2p/pull/4823).
```

Reply:
Wasn't sure which format to use, there were conflicting examples :slightly_smiling_face: